columba/profiles.py

```diff
@@ -24,7 +28,10 @@ def load(profile: str = "default"):
         buffer = stream.readlines()
         stream.close()
     except Exception:
-        return
+        raise ProfileNotFound(
+            f"'{profile}' not found. Please check the if profile '{profile}' "
+            "exists or create a new profile with 'aws.configure()'."
```

Comment:
Please check ~the~ if **the** profile...

E acho que eu removeria o início (`'{profile}' not found.`) por me parecer redundante.

Reply:
@julianolf o PR foi atualizado com os ajustes sugeridos.